packages/example-getting-started/docs/7-controller.md

```diff
@@ -1,14 +1,98 @@
+### Controllers
+
+In LoopBack 4, controllers handle the request-response lifecycle for your API.
+Each function on a controller can be addressed individually to handle
+an incoming request (like a POST request to `/todo`), perform business logic
+and then return a response.
+
+In this respect, controllers are the regions _in which most of your business
+logic will live_!
+
 ### Create your controller
 
-Now, we'll create a controller to handle our Todo routes. Create the
+So, let's create a controller to handle our Todo routes. Create the
 `src/controllers` directory and two files inside:
 - `index.ts` (export helper)
 - `todo.controller.ts`
 
-In addition to creating the CRUD methods themselves, we'll also be adding
+In addition to creating the handler functions themselves, we'll also be adding
 decorators that setup the routing as well as the expected parameters of
 incoming requests.
 
+First, we need to define our basic controller class as well as plug in our
+repository, which we'll use to perform our operations against the datasource.
+
+#### src/controllers/todo.controller.ts
+```ts
+import {repository} from '@loopback/repository';
+import {TodoRepository} from '../repositories/index';
+
+export class TodoController {
+  constructor(
+    @repository(TodoRepository.name) protected todoRepo: TodoRepository,
+  ) {}
+}
+```
+
+The `@repository` decorator will retrieve and inject an instance of the
+`TodoRepository` whenever an inbound request is being handled. The lifecycle
+of your controller object is typically a per-request lifecycle, which means that
```

Comment:
I was explaining it so that users wouldn't wonder why we're injecting the repository instead of instantiating it, which led into explaining why we usually instantiate the controller instead of injecting it. :P